server/client_manager.py

```diff
@@ -300,6 +300,11 @@ def send_timer_set_time(self, timer_id=None, new_time=None, start=False):
                 else:
                     self.send_command("TI", timer_id, 2, new_time) # Show timer
                     self.send_command("TI", timer_id, int(not start), new_time) # Set timer with value and start
+                    if timer_id == 0:
+                        timer = self.area.area_manager.timer
+                    else:
+                        timer = self.area.timers[timer_id-1]
+                    self.send_command("TF", timer_id, timer.format, new_time)
```

Comment:
does DRO support timer formatting or is this just a KFO-Client feature we came up with? I'll double-check.